src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -564,6 +564,11 @@ def launch_fluent(
             return new_session.create_from_server_info_file(
                 server_info_filepath, cleanup_on_exit, start_transcript
             )
+        except Exception as ex:
+            identifier = "fluent.exe"
+            print("Fluent Launch Path: ", launch_string.split(identifier)[0] + identifier)
```

Comment:
rather than printing, we can add this info to the exception, which guarantees it gets propagated in any context